src/services/productAPI.ts

```diff
@@ -0,0 +1,16 @@
+const handleError = (res: any) => {
+  if (!res.ok) {
+    throw Error(res.statusText);
+  }
+  return res.json();
+}
+
+export const productAPI = {
+  getProductByID: (product_id: string) => (
+    fetch(`https://reqres.in/api/users/${product_id}`)
```

Comment:
create http service with methods get, post, delete, put
use get there for https://desolate-lowlands-42565.herokuapp.com/products
api host https://desolate-lowlands-42565.herokuapp.com move to config